packages/cli/commands/import/workflow.ts

```diff
@@ -94,40 +105,83 @@ export class ImportWorkflowsCommand extends Command {
 				for (i = 0; i < files.length; i++) {
 					const workflow = JSON.parse(fs.readFileSync(files[i], { encoding: 'utf8' }));
 					if (credentialsEntities.length > 0) {
-						// eslint-disable-next-line
 						workflow.nodes.forEach((node: INode) => {
 							this.transformCredentials(node, credentialsEntities);
 						});
 					}
-					// eslint-disable-next-line no-await-in-loop, @typescript-eslint/no-non-null-assertion
 					await Db.collections.Workflow!.save(workflow);
+
+					const sharedWorkflow = new SharedWorkflow();
+
+					await Db.collections.SharedWorkflow!.save(
+						Object.assign(sharedWorkflow, {
+							user: owner,
+							workflow,
+							role: ownerWorkflowRole,
+						}),
+					);
```

Comment:
this should work, doesn't it? We should also think about making this operation a transaction or even inserting all 👍🏼 (I know we aren't doing it in other commands yet)
```suggestion
					await Db.collections.SharedWorkflow!.save({
							user: owner,
							workflow,
							role: ownerWorkflowRole,
						});
```